ui/webpack.common.ts

```diff
@@ -0,0 +1,94 @@
+const path = require('path')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const {CleanWebpackPlugin} = require('clean-webpack-plugin')
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const ForkTsCheckerWebpackPlugin = require('fork-ts-checker-webpack-plugin')
+const webpack = require('webpack')
+const GIT_SHA = require('child_process')
+  .execSync('git rev-parse HEAD')
+  .toString()
+
+const devMode = process.env.NODE_ENV !== 'production'
+
+module.exports = {
+  context: __dirname,
+  output: {
+    filename: '[name].bundle.js',
```

Comment:
We should probably add a hash in production.

Reply:
Good catch added